examples/webapp/build.gradle

```diff
@@ -7,11 +7,11 @@ publishMavenJavaPublicationToMavenRepository {
 }
 
 dependencies {
-    compile project(':core')
+    api project(':core')
```

Comment:
Technically, this is `implementation` I'd think.

Reply:
sure.